examples/machinedeployment/kustomization.yaml

```diff
@@ -5,3 +5,5 @@ resources:
 - machinedeployment.yaml
 configurations:
 - kustomizeconfig.yaml
+patchesStrategicMerge:
+- kustomizeversions.yaml
```

Comment:
Same comment her as with the controlplane kustomization.yaml

Reply:
as mentioned above. ... i can open a follow up issue